scripts/delete_released_changelogs.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# After a release is tagged, the individual changelog of the PRs are not
+# needed. This script is to delete the changelogs AFTER a release and is to
+# run as part of preparing the post release PR.
+#
+# This script accepts the newly released/tagged version as "--new_version"
+# and the previous version as "--prev_version" and will delete the
+# changelog files between them present in ".unreleased" folder.
+#
+# This script is run on the main branch and the changelog files deletion
+# is propogated to the release branch when the post release gets backported
+# to the release branch
+#
+
+import argparse
+import git
+import os
+
+# Changelogs directory
+CHANGE_LOGS_DIR = "./.unreleased"
+
+
+def delete_change_logs():
```

Comment:
Want to keep it in a file and so that this can be pointed from the release process documentation and any changes in this will not require a change in the doc.